src/Giveaway.js

```diff
@@ -369,42 +372,45 @@ class Giveaway extends EventEmitter {
 
     /**
      * Filles in a embed with giveaway properties.
-     * @param {Discord.MessageEmbed|Discord.MessageEmbedOptions} embed The embed that should get filled in.
-     * @returns {?Discord.MessageEmbed} The filled in embed.
+     * @param {EmbedBuilder|EmbedData|APIEmbed} embed The embed that should get filled in.
```

Comment:
does "EmbedData" still exist? cause i wasn't able to find it.
and the other the question is, if we leave `EmbedBuilder` or also go with JSONEncodable like dc.js docs. 
_idk if there is any other class that could be toJSONed as an embed_